src/cryptoadvance/specter/devices/bitcoin_core.py

```diff
@@ -0,0 +1,121 @@
+            { 'desc': AddChecksum('sh(wpkh({}/49h/1h/0h/0/*))'.format(testnet_xprv)), 'range': 1000, 'timestamp': 'now'},
+            { 'desc': AddChecksum('sh(wpkh({}/49h/1h/0h/1/*))'.format(testnet_xprv)), 'range': 1000, 'timestamp': 'now'},
+            { 'desc': AddChecksum('wpkh({}/84h/1h/0h/0/*)'.format(testnet_xprv)), 'range': 1000, 'timestamp': 'now'},
+            { 'desc': AddChecksum('wpkh({}/84h/1h/0h/1/*)'.format(testnet_xprv)), 'range': 1000, 'timestamp': 'now'},
+            { 'desc': AddChecksum('sh(wpkh({}/48h/1h/0h/1h/0/*))'.format(testnet_xprv)), 'range': 1000, 'timestamp': 'now'},
+            { 'desc': AddChecksum('sh(wpkh({}/48h/1h/0h/1h/1/*))'.format(testnet_xprv)), 'range': 1000, 'timestamp': 'now'},
+            { 'desc': AddChecksum('wpkh({}/48h/1h/0h/2h/0/*)'.format(testnet_xprv)), 'range': 1000, 'timestamp': 'now'},
+            { 'desc': AddChecksum('wpkh({}/48h/1h/0h/2h/1/*)'.format(testnet_xprv)), 'range': 1000, 'timestamp': 'now'},
+        ])
+        if passphrase:
+            cli.encryptwallet(passphrase)
+
+        bip32 = BIP32.from_seed(seed)
+        xpubs = ""
+        master_fpr = get_xpub_fingerprint(bip32.get_xpub_from_path('m/0h')).hex()
+        # Nested Segwit
+        xpub = bip32.get_xpub_from_path('m/49h/0h/0h')
+        ypub = convert_xpub_prefix(xpub, b'\x04\x9d\x7c\xb2')
+        xpubs += "[%s/49'/0'/0']%s\n" % (master_fpr, ypub)
+        # native Segwit
+        xpub = bip32.get_xpub_from_path('m/84h/0h/0h')
+        zpub = convert_xpub_prefix(xpub, b'\x04\xb2\x47\x46')
+        xpubs += "[%s/84'/0'/0']%s\n" % (master_fpr, zpub)
+        # Multisig nested Segwit
+        xpub = bip32.get_xpub_from_path('m/48h/0h/0h/1h')
+        Ypub = convert_xpub_prefix(xpub, b'\x02\x95\xb4\x3f')
+        xpubs += "[%s/48'/0'/0'/1']%s\n" % (master_fpr, Ypub)
+        # Multisig native Segwit
+        xpub = bip32.get_xpub_from_path('m/48h/0h/0h/2h')
+        Zpub = convert_xpub_prefix(xpub, b'\x02\xaa\x7e\xd3')
+        xpubs += "[%s/48'/0'/0'/2']%s\n" % (master_fpr, Zpub)
+        # Testnet nested Segwit
+        xpub = bip32.get_xpub_from_path('m/49h/1h/0h')
+        upub = convert_xpub_prefix(xpub, b'\x04\x4a\x52\x62')
+        xpubs += "[%s/49'/1'/0']%s\n" % (master_fpr, upub)
+        # Testnet native Segwit
+        xpub = bip32.get_xpub_from_path('m/84h/1h/0h')
+        vpub = convert_xpub_prefix(xpub, b'\x04\x5f\x1c\xf6')
+        xpubs += "[%s/84'/1'/0']%s\n" % (master_fpr, vpub)
+        # Testnet multisig nested Segwit
+        xpub = bip32.get_xpub_from_path('m/48h/1h/0h/1h')
+        Upub = convert_xpub_prefix(xpub, b'\x02\x42\x89\xef')
+        xpubs += "[%s/48'/1'/0'/1']%s\n" % (master_fpr, Upub)
+        # Testnet multisig native Segwit
+        xpub = bip32.get_xpub_from_path('m/48h/1h/0h/2h')
+        Vpub = convert_xpub_prefix(xpub, b'\x02\x57\x54\x83')
+        xpubs += "[%s/48'/1'/0'/2']%s\n" % (master_fpr, Vpub)
+        keys, failed = Key.parse_xpubs(xpubs)
```

Comment:
Makes sense to export only xpubs for the current network (only testnet or mainnet) as others will not be available anyways.